src/resources/ble_esp32.cc

```diff
@@ -608,11 +618,18 @@ class BlePeripheralManagerResource : public ServiceContainer<BlePeripheralManage
 class BleRemoteDeviceResource : public ServiceContainer<BleRemoteDeviceResource> {
  public:
   TAG(BleRemoteDeviceResource);
-  explicit BleRemoteDeviceResource(BleResourceGroup* group, bool secure_connection)
-    : ServiceContainer(group, REMOTE_DEVICE)
-    , handle_(kInvalidHandle)
-    , secure_connection_(secure_connection)
-    , connected_(false) {}
+  explicit BleRemoteDeviceResource(BleResourceGroup* group, BleCentralManagerResource* central_manager, bool secure_connection)
```

Comment:
No need for explicit, but it's fine to keep it.

Reply:
Keeping it, as it doesn't hurt.